spec/features/user_logs_in_spec.rb

```diff
@@ -25,33 +25,57 @@
       visit '/login'
       fill_in('session[email]', with: 'example@example.com')
       click_link_or_button 'Login'
-      within("#errors") do
+      within(".flash") do
         expect(page).to have_content("Invalid Login")
       end
     end
 
-  end
-
-  context 'as a valid default user' do
-    let(:user) { User.create(email: 'example@example.com', password: 'password') }
-
-    scenario 'can log in' do
-      user
+    scenario 'logging in does not clear the cart' do
+      user = User.create(full_name: 'dg',
+                             email: 'example@example.com',
+                          password: 'password')
+      create(:item)
+      visit '/menu'
+      within("#items") do
+        click_link_or_button 'Add To Cart'
+      end
+      visit '/cart'
+      expect(page).to have_content("Sushi")
+      expect(page).to_not have_content("Onigiri")
+      expect(page).to have_content("1")
       visit '/login'
       fill_in('session[email]', with: user.email)
       fill_in('session[password]', with: user.password)
       click_link_or_button 'Login'
       expect(page).to have_content("You are logged in")
+      visit '/cart'
+      expect(page).to have_content("Sushi")
+      expect(page).to_not have_content("Onigiri")
+      expect(page).to have_content("1")
+    end
+  end
 
+  context 'as a valid default user' do
+    let(:user) { User.create(full_name: "dg",
+                              email: "example@example.com",
+                              password: "password") }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 61, 53 should be on its own line.